src/lib/exportSave.ts

```diff
@@ -77,12 +86,15 @@ export async function exportSave(data: ArrayBuffer, toastId: string) {
   if (files.length > 1) {
     let zip = new JSZip()
     for (const file of files) {
-      zip.file(file.name, new Uint8Array(file.contents), { binary: true })
+      zip.file(fileName || file.name, new Uint8Array(file.contents), {
```

Comment:
we should keep the internals of the zip named the same, not replace it, because there could be references and we do not want to break them, I'd just change the name of the zip